api/mutations/thread/publishThread.js

```diff
@@ -55,253 +55,254 @@ type Input = {
   },
 };
 
-export default requireAuth(async (_: any, args: Input, ctx: GraphQLContext) => {
-  const { user, loaders } = ctx;
-  const { thread } = args;
+export default requireAuth(
+  async (_: any, args: PublishThreadInput, ctx: GraphQLContext) => {
```

Comment:
I really don't know why it reformatted this entire file, but the only thing I changed here was rename the type to PublishThreadInput and export it.